src/frontend/Felles/Personopplysninger/TabellWrapper.tsx

```diff
@@ -7,10 +7,6 @@ export const BredTd = styled.td`
     padding-left: 0;
 `;
 
-export const Th = styled.td`
```

Comment:
Fjernet styling for Th, men ikke Td - er det meningen?

Reply:
Td brukes i en annen tabell i løsningen - Bostedsadresse.tsx